GA-worksheet.md

```diff
@@ -6,18 +6,18 @@
 
 
 ##Project #2 Part #1: Header HTML
-1. What is the purpose of the header tag?
+1. What is the purpose of the header tag? The purpose of the header tag is to hold up the information in the body of the website such as the image. 
```

Comment:
:+1:  More specifically, the header tag actually holds the introductory material for the website, such as a logo, and navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify).